packages/fxa-settings/src/components/App/index.tsx

```diff
@@ -81,24 +81,72 @@ import ResetPasswordContainer from '../../pages/ResetPasswordRedesign/ResetPassw
 import ConfirmResetPasswordContainer from '../../pages/ResetPasswordRedesign/ConfirmResetPassword/container';
 import CompleteResetPasswordWithCodeContainer from '../../pages/ResetPasswordRedesign/CompleteResetPassword/container';
 import AccountRecoveryConfirmKeyContainer from '../../pages/ResetPasswordRedesign/AccountRecoveryConfirmKey/container';
+import SignoutSync from '../../pages/SignoutSync';
 
 const Settings = lazy(() => import('../Settings'));
 
 export const App = ({
   flowQueryParams,
 }: { flowQueryParams: QueryParams } & RouteComponentProps) => {
   const config = useConfig();
+  const session = useSession();
+  const integration = useIntegration();
+  const isSync = integration != null && integration.isSync();
```

Comment:
Just noting, this will be `true` for any Sync integrations. Previously, this was targeting mobile sync accessing Settings only.

Reply:
It doesn't seem like mobile should be a special case. After testing this more, it seems like there are different ways one could end the session without FF knowing about it, and despite it being easy to do on android, I don't think it'd be impossible on desktop.

Also, I dropped the check on config.sendFxAStatusOnSettings. This flag seemed liked it muddied the waters as well.

Reply:
> It doesn't seem like mobile should be a special case. After testing this more, it seems like there are different ways one could end the session without FF knowing about it, and despite it being easy to do on android, I don't think it'd be impossible on desktop.

Yeah, sounds OK to me, just wanted to call it out. We should be running `requestSignedInUser` on `app-start` and that `mergeBrowserAccount` function, but apparently there is a bug around it right now (bug linked in that #sync Slack thread too).

> Also, I dropped the check on config.sendFxAStatusOnSettings. This flag seemed liked it muddied the waters as well.

👍 Vijay and I meant to remove this and didn't get to it.